app/models/organization.rb

```diff
@@ -133,44 +133,61 @@ def self.query(query, options = {})
   end
 
   def self.parse_message(message)
+    geonames = message.dig("locations", 0, "geonames_details")
```

Comment:
So this always grabs the first location in the array if present.
Is this a preference that we document?